Sources/SimultaneouslyScrollView/Extensions/MulticastScrollViewDelegate.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit;
```

Comment:
Please add the `#if os(iOS) || os(tvOS) || os(visionOS)` to rule out build errors on other platforms (macOS)